sklearn/cluster/_optics.py

```diff
@@ -231,8 +231,8 @@ def fit(self, X, y=None):
 
         Parameters
         ----------
-        X : array, shape (n_samples, n_features), or (n_samples, n_samples)  \
-        if metric=’precomputed’
+        X : array of shape (n_samples, n_features), or \
+        (n_samples, n_samples) if metric=’precomputed’
```

Comment:
Same here (`ndarray` and over-indented for visual indent).

Reply:
ok